src/main/java/biblioteka/Calendar.java

```diff
@@ -0,0 +1,354 @@
+package biblioteka;
+import model.Thing;
+import model.User;
+import java.sql.*;
+import java.util.List;
+//https://www.postgresqltutorial.com/postgresql-jdbc/ - fajny tutorial, sa tam tez metody do update'u czy kasowania ale narazie niepotrzebne
+//https://www.postgresqltutorial.com/postgresql-jdbc/query/ - na podstawie tego wyszukiwanie Userow zrobione
+//https://www.postgresqltutorial.com/postgresql-jdbc/insert/ - na podstawie tego dodawanie Userow zrobione
+
+public class Calendar {
+
+    private static String URL = "jdbc:postgresql://localhost:8012/event_base";
+    private static String USER = "postgres";
+    private static String PASSWORD = "password";
+
+    private Connection conn;
```

Comment:
Please use full names. There is enough RAM for every long name. :D `databaseConnection`?

Reply:
Done. Renamed conn to databaseConnection and stat to sqlStatement